src/main/java/org/usfirst/frc/team4099/robot/Constants.kt

```diff
@@ -22,6 +36,23 @@ class Constants {
         val RIGHT_SLAVE_1_ID = 5
         val RIGHT_SLAVE_2_ID = 0
 
+        val HIGH_GEAR_MAX_SETPOINT = 17.0  //17 fps
+    }
+
+    object Wheels {
+        val DRIVE_WHEEL_DIAMETER_INCHES = 0
+    }
+
+    object Velocity {
+        val HIGH_GEAR_VELOCITY_CONTROL_SLOT = 0
+        val LOW_GEAR_VELOCITY_CONTROL_SLOT = 0
+        val DRIVE_HIGH_GEAR_NOMINAL_OUTPUT = 0.0 //percentage
+        val DRIVE_LOW_GEAR_NOMINAL_OUTPUT = 0.0 //percentage
+        val DRIVE_HIGH_GEAR_FORWARD_OUTPUT = 1.0 //percentage
```

Comment:
Rename to `DRIVE_HIGH_GEAR_MAX_FORWARD_OUTPUT` or something similar to clarify that it's a max